### ISSUE TITLE
When filtering on epics, avoid filter combo's that don't make sense

### ISSUE THREAD

**Issue by adamreisnz** [ZenHubIO/support#976]
I was filtering on a specific epic, and then I clicked "Show all issues not in an epic". This resulted in zero issues being shown, as the specific epic was still toggled:

![image](https://cloud.githubusercontent.com/assets/490562/20908478/075b30f0-bbba-11e6-91cd-18123968d756.png)

Obviously these two selections are mutually exclusive, so the filter should probably be smart about it and unselect any incompatible selections.

**Comment by azenMatt** [comment]
Thanks for the feedback @adamreisnz - agreed this one is a bug!

**Comment by rashagen** [comment]
Would love to see this one fixed.

**Comment by mpittkin** [comment]
Also, if you select multiple epics, it must be using AND logic, because it doesn't show any issues at all. This should be fixed togther, just use OR logic when evaluating the epic filters, including issues not in an epic.

**Comment by adamreisnz** [comment]
Yeah they do that with releases too (AND filter instead of OR), which is bugging me a lot, because rarely (if at all) do we put issues in more than one release or epic, and an OR filter would be much more useful.
